src/audio/google_audio_post_processing.c

```diff
@@ -0,0 +1,551 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Google LLC.
```

Comment:
I think it will be worth thinking on creating a separate vendor directory in audio for each specific component. 

E.g ```/src/audio/google/```

Reply:
Agreed, I will do that as a separate PR after this lands